### DISCUSSION TITLE
A BIG documentation issue you guys should fix

### DISCUSSION THREAD

**Discussion by ghost** [hydro-dev/Hydro#214] [General]
When I use yarn 3.0.4, `yarn global` is removed.( in my case )
So you should tell the users which version of yarn should they use in the tutorial.

**Comment by undefined-moe** [comment]
`npm i yarn -g` will install yarn version 1.

**Reply by ghost** [reply]
Oh.OK.

**Reply by undefined-moe** [reply]
In hydro-project root, yarn 2 was installed locally. That's not the problem as if you are running in dev mode, you should always add your plugin, deps into the local workspace, not global storage.

**Reply by ghost** [reply]
By the way, is `yarn debug` also the command to restart the website?

**Reply by undefined-moe** [reply]
`yarn debug` is for starting backend in development mode.
If you install Hydro globally, you can simply use `hydrooj` to start the application.
If you are using install script, you can restart the app using `pm2 restart hydrooj`

**Reply by ghost** [reply]
Another question: If I modified `main.html` to show extra cards, (in develop mode), how can I restart?
Found one solution:
kill the proc.
Is there other easy ways?

**Reply by undefined-moe** [reply]
I think if you start with `yarn debug` and you can see the changes applied without restarting.
Note: it has an impact on performance. DO NOT use debug mode in production.

**Reply by ghost** [reply]
I'll try.
